source/type/pb_converter.cpp

```diff
@@ -1849,6 +1855,16 @@ std::string PbCreator::get_section_id(pbnavitia::Journey* j, size_t section_idx)
     return it->second;
 }
 
+std::string PbCreator::get_translated_message(std::vector<type::disruption::Translation> translations,
```

Comment:
```suggestion
std::string PbCreator::get_translated_message(std::vector<type::disruption::Translation> & translations,
```
You don't want to copy the vector when calling this function !